changelog.d/pr-9346

```diff
@@ -0,0 +1,13 @@
+synopsis: remove 
+packages: cabal-install
+prs: #9346
+issues: #9151
+significance: significant
+
+description: {
+
+- Log is used in cabal-install as a loglevel.
+- "Log" is not a [loglevel](https://betterstack.com/community/guides/logging/log-levels-explained/).
+- replace Log with Info where it is used.
```

Comment:
```suggestion
Remove "Log" as a log level in favour of "Info".
```

I think a simple entry like this is enough. No need to lecture users in a changelog :D (although I'd be happy to be lectured personally, I bookmarked the link).

Reply:
I apologize for the tone of how it came out: English isn't my native.

I'm most happy being lectured on how to make better contributions.